apps/aeutils/priv/epoch_config_schema.json

```diff
@@ -417,42 +476,45 @@
                                 "executable_group" : {
                                     "description" : "Group of executable binaries of the miner.",
                                     "type" : "string",
-                                    "enum" : [ "aecuckoo", "aecuckooprebuilt" ],
-                                    "default": "aecuckoo"
+                                    "enum" : [
+                                        "aecuckoo",
+                                        "aecuckooprebuilt"
+                                    ],
+                                    "default" : "aecuckoo"
                                 },
                                 "executable" : {
                                     "description" : "Executable binary of the miner. If the executable group is 'aecuckoo', the options are: \"mean29-generic\" (memory-intensive), \"mean29-avx2\" (memory-intensive, benefits from faster CPU supporting AVX2 instructions), \"lean29-generic\" (CPU-intensive, useful if memory-constrained), \"lean29-avx2\" (CPU-intensive, useful if memory-constrained, benefits from faster CPU supporting AVX2 instructions). For the other executable groups, please refer to the user documentation.",
                                     "type" : "string",
-                                    "default": "mean29-generic"
+                                    "default" : "mean29-generic"
                                 },
                                 "extra_args" : {
                                     "description" : "Extra arguments to pass to the miner executable binary. The safest choice is specifying no arguments i.e. empty string.",
                                     "type" : "string",
-                                    "default": "-t 1"
+                                    "default" : ""
```

Comment:
This is the only change in previous behaviour that I found based on code inspection.